frontend/src/page/profile.tsx

```diff
@@ -26,6 +28,44 @@ export const Profile: React.FC = () => {
   }
   return (
     <div>
```

Comment:
If you don't need the extra <div>s I would remove them (unless they're for spacing or something)

Reply:
Yeah, shouldn't need them now that I'm using the material-ui components. :ok_man: